goblin/abc.py

```diff
@@ -46,9 +46,37 @@ def to_ogm(self, val):
         """Convert property value to a Python compatible format"""
         return val
 
+    def validate_cardinality(self, val, cardinality):
+        if cardinality:
+            if issubclass(cardinality, list):
```

Comment:
Maybe we could use constants for cardinality instead of collection classes? E.g. `cardinality.SINGLE`, `cardinality.LIST`, `cardinality.SET`, where `cardinality` is either a module or object. In gremlin-java / gremlin-groovy it's an enum.